auditlog/models/rule.py

```diff
@@ -400,6 +435,16 @@ def write_fast(self, vals, **kwargs):
             old_values = {id_: old_vals2 for id_ in self.ids}
             new_values = {id_: vals2 for id_ in self.ids}
             result = write_fast.origin(self, vals, **kwargs)
+            rule = rule_model.sudo().search([("model_id.model", "=", self._name)])
```

Comment:
same + reinforce the fact that w/ a specific method you don't repeat this code everywhere